### ISSUE TITLE
Show or Hide Addend buttons need unique names - accessible name issue

### ISSUE THREAD

**Issue by terracoda** [phetsims/number-pairs#153]
The icons communicate a single state.
The current names, "Show or Hide Addend" communicate two actions and the names are identical for the both sides of the board.

If you want to communicate the same information as the icon is showing, I would suggest using "Visible" and "Hidden" rather than the actions of "Show" and "Hide." 

BUT since the names are not visible, I think an action-oriented name would sound better, especially when you add Tier 2 response that might be "{{Apples}} on left hidden," or something like that. 

Additionally, you need location "Left" and "Right" to make the accessible names unique, AND
like the "Organize" button, I think it makes sense to name the representations.

Here are some possibilities depending on how you what you want to communicate - and action or a state.

![Image](https://github.com/user-attachments/assets/42477cc1-c565-46a0-96c3-a7b723615545)
"Hide {{Apples}} on Left" or "{{Apples}} on Left, Visible"
"Hide Count on Left" or "Count on Left, Visible"
"Hide Left Counting Area" or "Left Counting Area, Visible"
"Hide Left Side" or "Left Side, Visible"
"Hide Left Addend" or "Left Addend, Visible"

![Image](https://github.com/user-attachments/assets/82acaf1e-be48-444c-9803-6edbe96c8e9e)
"Show {{Apples}} on Left" or "{{Apples}} on Left, Hidden"
"Show Count on Left" or "Count on Left, Hidden"
"Show Left Counting Area" or "Left Counting Area, Hidden"
"Show Left Side" or "Left Side, Hidden"
"Show Left Addend" or "Left Addend, Hidden"

**Comment by marlitas** [comment]
My understanding is because it is a toggle button I did not need to specify "hide" vs. "show"

From https://github.com/phetsims/number-pairs/issues/78#issuecomment-2632084467

> Question for MS: Is this implemented as a toggle button? If so, the state will say whether the button is pressed or not pressed. If not, then use the names on the left For location-based: * Show or hide the objects and the value of the addend. For other: * Show or hide the objects and values for both addends.

**Comment by marlitas** [comment]
We decided to do ("hide" when visible, "show" when hidden):
- "Hide Left Addend" vs. "Show Left Addend"
- "Hide Right Addend" vs. "Show Left Addend"
- "Hide Both Addends" vs. "Show Both Addends"

**Comment by marlitas** [comment]
Updated above. @catherinecarter and @terracoda feel free to close if all looks well.

**Comment by catherinecarter** [comment]
The description of the buttons is as described above and working well for the Intro, Ten, and Twenty screens.

I have a question about the Sum screen. When the objects are visible, the a11y view is as shown: 
![Image](https://github.com/user-attachments/assets/12186504-fb77-46d8-88a4-969c6edccede)

When I hide the objects, the a11y view looks this this: 
![Image](https://github.com/user-attachments/assets/a567ab28-4fa9-4955-a991-980d7e9fe431)

My question is whether the values of the addends will be read since they're in the a11y view, even though the `countingArea` is hidden? I'm not familiar with screen readers, so I'm not sure what will be voiced here.

**Comment by terracoda** [comment]
This issue is related to #78

**Comment by catherinecarter** [comment]
I think you're correct. Do you know the answer to my question, though?

> My question is whether the values of the addends will be read since they're in the a11y view, even though the countingArea is hidden?

**Comment by terracoda** [comment]
The button names makes sense now. Thanks.

**Comment by terracoda** [comment]
@catherinecarter, sorry I missed your question. The numbers on the "Object" number spinners will be read as they remain accessible through the aria-valuetext on the spinner. If you want to hide that number as well (makes sense), you would the toggle to control both the counting area and the spinners. 

Right now, with the current state of the design, the number spinners are the only control providing the addend values. I imagine in the future the counting area might have some addend information associated with it.

Again, if you want to hide the addend in both modalities the toggle button has to watch all representations, both in the visual view as well as the descriptions of those representations.
